tests/Assets/SimpleEnumPhp81.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineTest\Laminas\Hydrator\Assets;
+
+enum SimpleEnumPhp81: int
+{
+case One = 1;
+case Two = 2;
+    }
```

Comment:
Alignment is still very weird :D

Reply:
Indeed. I had it fixed, but then ran `composer cs-fix` afterwards. Just tried to reproduce it, indeed PHPCSFixer introduces this alignment. :-\

Reply:
Eh, it's fine then, let's leave it to not fight the tooling too much: it's a test asset anyway.